src/tests/ftest/util/server_utils.py

```diff
@@ -122,30 +130,46 @@ def create_server_yaml(basepath):
         raise ServerFailed("Failed to Write {}/{}".format(basepath,
                                                           AVOCADO_FILE))
 
-def run_server(hostfile, setname, basepath, uri_path=None, env_dict=None):
-    """
-    Launches DAOS servers in accordance with the supplied hostfile.
+
+def run_server(hostfile, setname, basepath, uri_path=None, env_dict=None,
+               clean=True):
+    """Launch DAOS servers in accordance with the supplied hostfile.
+
+    Args:
+        hostfile (str): hostfile defining on which hosts to start servers
+        setname (str): session name
+        basepath (str): DAOS install basepath
+        uri_path (str, optional): path to uri file. Defaults to None.
+        env_dict (dict, optional): dictionary on env variable names and values.
+            Defaults to None.
+        clean (bool, optional): remove files in /mnt/daos. Defaults to True.
+
+    Raises:
+        ServerFailed: if there is an error starting the servers
+
     """
-    global SESSIONS
+    global SESSIONS    # pylint: disable=global-variable-not-assigned
     try:
         servers = (
             [line.split(' ')[0] for line in genio.read_all_lines(hostfile)])
         server_count = len(servers)
 
-        #Create the DAOS server configuration yaml file to pass
-        #with daos_server -o <FILE_NAME>
+        # Create the DAOS server configuration yaml file to pass
+        # with daos_server -o <FILE_NAME>
         create_server_yaml(basepath)
 
-        # first make sure there are no existing servers running
+        # First make sure there are no existing servers running
         kill_server(servers)
 
-        # clean the tmpfs on the servers
-        for server in servers:
-            subprocess.check_call(['ssh', server,
-                                   ("find /mnt/daos -mindepth 1 -maxdepth 1 "
-                                    "-print0 | xargs -0r rm -rf")])
+        # Clean the tmpfs on the servers
+        if clean:
+            for server in servers:
```

Comment:
O clush, clush, wherefore art thou clush?

Reply:
Yes, using clush would be better and result in speed bump for all tests, but this should be a separate PR.  I'll create one.